src/js/msp/MSPHelper.js

```diff
@@ -1095,69 +1095,71 @@ MspHelper.prototype.process_data = function(dataHandler) {
                 ADVANCED_TUNING_ACTIVE = { ...ADVANCED_TUNING };
                 break;
             case MSPCodes.MSP_PID_ADVANCED:
-                ADVANCED_TUNING.rollPitchItermIgnoreRate = data.readU16();
-                ADVANCED_TUNING.yawItermIgnoreRate = data.readU16();
-                ADVANCED_TUNING.yaw_p_limit = data.readU16();
-                ADVANCED_TUNING.deltaMethod = data.readU8();
-                ADVANCED_TUNING.vbatPidCompensation = data.readU8();
-                if (semver.gte(CONFIG.apiVersion, "1.20.0")) {
-                    if (semver.gte(CONFIG.apiVersion, "1.40.0")) {
-                        ADVANCED_TUNING.feedforwardTransition = data.readU8();
-                    } else {
-                        ADVANCED_TUNING.dtermSetpointTransition = data.readU8();
-                    }
-                    ADVANCED_TUNING.dtermSetpointWeight = data.readU8();
-                    ADVANCED_TUNING.toleranceBand = data.readU8();
-                    ADVANCED_TUNING.toleranceBandReduction = data.readU8();
-                    ADVANCED_TUNING.itermThrottleGain = data.readU8();
-                    ADVANCED_TUNING.pidMaxVelocity = data.readU16();
-                    ADVANCED_TUNING.pidMaxVelocityYaw = data.readU16();
-                    if (semver.gte(CONFIG.apiVersion, "1.24.0")) {
-                        ADVANCED_TUNING.levelAngleLimit = data.readU8();
-                        ADVANCED_TUNING.levelSensitivity = data.readU8();
-
-                        if (semver.gte(CONFIG.apiVersion, "1.36.0")) {
-                            ADVANCED_TUNING.itermThrottleThreshold = data.readU16();
-                            ADVANCED_TUNING.itermAcceleratorGain = data.readU16();
-
-                            if (semver.gte(CONFIG.apiVersion, "1.39.0")) {
-                                ADVANCED_TUNING.dtermSetpointWeight = data.readU16();
-
-                                if (semver.gte(CONFIG.apiVersion, "1.40.0")) {
-                                    ADVANCED_TUNING.itermRotation = data.readU8();
-                                    ADVANCED_TUNING.smartFeedforward = data.readU8();
-                                    ADVANCED_TUNING.itermRelax = data.readU8();
-                                    ADVANCED_TUNING.itermRelaxType = data.readU8();
-                                    ADVANCED_TUNING.absoluteControlGain = data.readU8();
-                                    ADVANCED_TUNING.throttleBoost = data.readU8();
-                                    ADVANCED_TUNING.acroTrainerAngleLimit = data.readU8();
-                                    ADVANCED_TUNING.feedforwardRoll  = data.readU16();
-                                    ADVANCED_TUNING.feedforwardPitch = data.readU16();
-                                    ADVANCED_TUNING.feedforwardYaw   = data.readU16();
-                                    ADVANCED_TUNING.antiGravityMode  = data.readU8();
+                if (semver.gte(CONFIG.apiVersion, "1.16.0")) {
```

Comment:
Usually you will not see the "base" MSP version of each command, because when sent it will simply return "command not valid", but it will not break nothing.
Is not bad to add it, but is not necessary.

Reply:
Yes, I just finding the coherency with the writing part, here is defined Api version but not for read